framework/src/modules/chain/submodules/blocks/chain.js

```diff
@@ -488,7 +488,8 @@ __private.loadSecondLastBlockStep = function(secondLastBlockId, tx) {
 			{ id: secondLastBlockId },
 			(err, blocks) => {
 				if (err || !blocks.length) {
-					library.logger.error('Failed to get loadBlocksPart', err);
+					const error = Array.isArray(err) && err.length > 0 ? err[0] : err;
+					library.logger.error('Failed to get loadBlocksPart', error);
 					return setImmediate(reject, err || 'previousBlock is null');
```

Comment:
'previousBlock is null' should be wrapped inside an error object as well. Right?